README.md

```diff
@@ -1,16 +1,40 @@
-# Release Integration Tests
+# Release Integration Test
 
+This README provides instructions on running a test using the provided shell script.
 
-## 1. FBC Test (happy path):
-The purpose of this release test (FBC) is to verify the functionality and stability of feature in RHTAP.
-Test is a "happy-path" test and it's running on the internal RHTAP stage cluster.
+## Test Steps and Expected Result
+
+1. **Pre-Running Setup:** The following setup actions need to be completed before running the test script.
+   
+2. **Running the Test:**
+   - The test script sets up resources, waits for pipeline runs to complete, checks release status, and reports the result.
```

Comment:
Why is this bulleted but nothing is bulleted for 1. or 3.?

Reply:
nice catch , 
by mistake.
removed